EpoxyLayoutGroups.podspec

```diff
@@ -0,0 +1,9 @@
+Pod::Spec.new do |spec|
+  # Update ConfigurePodspec.rb to increment the version number.
+  require_relative 'ConfigurePodspec'
+  configure(
+    spec: spec,
+    name: 'EpoxyLayoutGroups',
+    summary: 'Declarative API for driving the subviews of a UIView',
```

Comment:
It might be nice to include some details about how this supports nesting/is composable since it allows creating most layouts. Also the flattening logic is novel but not sure if you want to mention that.